pandas/core/dtypes/dtypes.py

```diff
@@ -74,7 +74,7 @@ def register(self, dtype: Type[ExtensionDtype]) -> None:
 
     def find(
         self, dtype: Union[Type[ExtensionDtype], str]
-    ) -> Optional[Type[ExtensionDtype]]:
+    ) -> Optional[Union[ExtensionDtype, Type[ExtensionDtype]]]:
```

Comment:
why is this changed?

Reply:
construct_from_string on L98 returns an instance. However, since the type annotations for construct_from_string were removed following https://github.com/pandas-dev/pandas/pull/31352#discussion_r371590898, this can also be removed for now.